src/ocm/components/featureSupportLevels/FeatureSupportLevelProvider.tsx

```diff
@@ -137,20 +141,30 @@ export const FeatureSupportLevelProvider: React.FC<SupportLevelProviderProps> =
     [getDisabledReasonCallback],
   );
 
+  const isFeatureFullySupportedCallback = React.useCallback(
+    (versionName: string, featureId: FeatureId) => {
+      const supportLevel = getFeatureSupportLevel(versionName, featureId);
+      return isFeatureFullySupported(versionName, featureId, supportLevel, versionOptions);
```

Comment:
Do we really need the `isFeatureFullySupported`?
Since you already have the `supportLevel`, you could just compare it directly.

(I'm ignoring the condition for ARM since it looks like too specific and we wouldn't be using this function with this feature)

Reply:
True, I've changed it